core/src/main/java/ai/timefold/solver/core/api/domain/common/SorterWeightFactory.java

```diff
@@ -0,0 +1,31 @@
+package ai.timefold.solver.core.api.domain.common;
+
+import ai.timefold.solver.core.api.domain.entity.PlanningEntity;
+import ai.timefold.solver.core.api.domain.solution.PlanningSolution;
+import ai.timefold.solver.core.impl.heuristic.move.Move;
+import ai.timefold.solver.core.impl.heuristic.selector.Selector;
+
+/**
+ * Creates a weight to decide the order of a collections of selections
+ * (a selection is a {@link PlanningEntity}, a planningValue, a {@link Move} or a {@link Selector}).
+ * The selections are then sorted by their weight,
+ * normally ascending unless it's configured descending.
```

Comment:
Configured how, where?

Reply:
The `sortManner` field in the value and entity selector configuration specifies how the data will be sorted. There is also a setting called `constructionHeuristicType` that automatically configures how entities and values are sorted.

Reply:
Don't tell me, tell the users. :-)